pkg/autodeployer/autodeployer.go

```diff
@@ -0,0 +1,262 @@
+package autodeployer
+
+import (
+	"sort"
+	"sync"
+
+	"github.com/blang/semver"
+	"github.com/pkg/errors"
+	downstreamtypes "github.com/replicatedhq/kots/pkg/api/downstream/types"
+	apptypes "github.com/replicatedhq/kots/pkg/app/types"
+	"github.com/replicatedhq/kots/pkg/logger"
+	"github.com/replicatedhq/kots/pkg/store"
+	storetypes "github.com/replicatedhq/kots/pkg/store/types"
+	"github.com/replicatedhq/kots/pkg/version"
+	cron "github.com/robfig/cron/v3"
+	"go.uber.org/zap"
+)
+
+// jobs maps app ids to their cron jobs
+var jobs = make(map[string]*cron.Cron)
+var mtx sync.Mutex
+
+// Start will start the auto deployer
+// the frequency of those update checks are app specific and can be modified by the user
+func Start() error {
+	logger.Debug("starting auto deployer")
+
+	appsList, err := store.GetStore().ListInstalledApps()
+	if err != nil {
+		return errors.Wrap(err, "failed to list installed apps")
+	}
+
+	for _, a := range appsList {
+		if a.IsAirgap {
```

Comment:
`Configure` also checks this

Reply:
which is fine, can't be too careful :). configure is called by other packages as well

Reply:
i can get rid of it